strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RegenerateNugetChecksumInStoragesCronJobTestIT.java

```diff
@@ -0,0 +1,145 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import org.carlspring.strongbox.artifact.coordinates.NugetArtifactCoordinates;
+import org.carlspring.strongbox.config.NugetLayoutProviderCronTasksTestConfig;
+import org.carlspring.strongbox.providers.io.RepositoryFiles;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.providers.io.RootRepositoryPath;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.storage.repository.RepositoryPolicyEnum;
+import org.carlspring.strongbox.testing.artifact.ArtifactManagementTestExecutionListener;
+import org.carlspring.strongbox.testing.artifact.NugetTestArtifact;
+import org.carlspring.strongbox.testing.repository.NugetRepository;
+import org.carlspring.strongbox.testing.storage.repository.RepositoryManagementTestExecutionListener;
+import org.carlspring.strongbox.util.ThrowingConsumer;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.collect.Maps;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.Execution;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import static java.nio.file.Files.deleteIfExists;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.parallel.ExecutionMode.CONCURRENT;
+
+/**
+ * @author Kate Novik.
+ * @author Pablo Tirado
+ */
+@ContextConfiguration(classes = NugetLayoutProviderCronTasksTestConfig.class)
+@SpringBootTest(properties = { "strongbox.vault=target/strongbox-vault-rnciscjtit" })
```

Comment:
@carlspring have you tried `"strongbox.vault=${strongbox.basedir}/strongbox-vault-rnccjt"` as suggested?

Reply:
That creates a directory in the base dir of the module. It's also not named according to the test's abbreviated name. Also, wouldn't this conflict with the other test's abbreviated name?

Reply:
I mean we should try to use `${strongbox.basedir}` placeholder instead of hardcoded `target` folder.

Reply:
As I said, that places it under `strongbox/strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/strongbox-vault-rnciscjtit` which is then never cleaned. When it comes to tests, `${strongbox.basedir` would evaluate to `.`, which would resolve to `strongbox/strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider`. In our case, we want this to be wiped out next time the test runs, which is why I did it like this.

If you have a better idea, I'm all ears. :)

Reply:
could you try `${strongbox.home}/../strongbox-vault-rnciscjtit` ? The `strongbox.home` devined as `<strongbox.home>${project.build.directory}/strongbox</strongbox.home>` in `pom.xml` so this way it should be in `target`.

Reply:
Done.